src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,45 @@
-export const PeopleFilters = () => {
+import cn from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+import { SearchLink } from './SearchLink';
+import { getSearchWith } from '../utils/searchHelper';
+
+const centuries = ['16', '17', '18', '19', '20'];
+
+export const PeopleFilters: React.FC = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex') || null;
+  const centuryParams = searchParams.getAll('centuries') || [];
+
+  const handleSearchInput = (e: React.ChangeEvent<HTMLInputElement>) => {
+    const search = getSearchWith(searchParams, { query: e.target.value });
+
+    setSearchParams(search);
+  };
+
   return (
     <nav className="panel">
       <p className="panel-heading">Filters</p>
 
       <p className="panel-tabs" data-cy="SexFilter">
-        <a className="is-active" href="#/people">All</a>
-        <a className="" href="#/people?sex=m">Male</a>
-        <a className="" href="#/people?sex=f">Female</a>
+        <SearchLink
+          params={{ sex: null }}
+          className={cn({ 'is-active': !sex })}
+        >
+          All
+        </SearchLink>
+        <SearchLink
+          params={{ sex: 'm' }}
+          className={cn({ 'is-active': sex === 'm' })}
```

Comment:
m is a 'magic word'
Consider will be better if you create a enum for magic words and use it everywhere